parts/openshift/release-3.9/openshiftmasterscript.sh

```diff
@@ -31,13 +31,6 @@ else
 	COCKPIT_VERSION="latest"
 fi
 
-# TODO: with WALinuxAgent>=v2.2.21 (https://github.com/Azure/WALinuxAgent/pull/1005)
-# we should be able to append context=system_u:object_r:container_var_lib_t:s0
-# to ResourceDisk.MountOptions in /etc/waagent.conf and remove this stanza.
-systemctl stop docker.service
```

Comment:
I believe the waagent changes to format and use /dev/sdb1 never made it into the 3.9 image so I believe this code was redundant?